File: command-design/light.py
class Light:
    def __init__(self, name: str):
        self.isOn = False
        self.name = name
    
    def turnOn(self):
        self.isOn = True
        print(f"Light {self.name} turned on")
    
    def turnOff(self):
        self.isOn = False
        print(f"Light {self.name} turned off")

File: command/Light/Light.java
package command.Light;

public class Light {
    public boolean on;

    public Light() {
        this.on = false;
    }

    public void turnOn() {
        if (this.on == true) {
            System.out.println("Light already on.");
        } else {
            System.out.println("Turned on the light.");
            this.on = true;
        }
    }

    public void turnOff() {
        if (this.on == false) {
            System.out.println("Light already off.");
        } else {
            System.out.println("Turned off the light.");
            this.on = false;
        }
    }
}
